roles/ceph-common/tasks/facts.yml

```diff
@@ -17,6 +17,9 @@
 - set_fact:
     init_system={{ init_system.content | b64decode }}
 
+- set_fact:
+    use_systemd={{ init_system == 'systemd\n' }}
```

Comment:
I don't like including the '\n' here, but I couldn't find a good filter provided by jinja2 to remove it. The logs in the CI tests proved it's needed though.